### PR TITLE
Allow filtering of "removing", it is a valid status

### PR DESCRIPTION
**Pull request by rhatdan**
Do not use a list of statuses outside of libpod to validate container
statuses.  Removing status was never added to the list.

Fixes: https://github.com/containers/podman/issues/13986

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@[nivekuil](https://github.com/nivekuil) PTAL
@containers/podman-desktop-maintainers PTAL

**Review by benoitf — COMMENTED** [review]
👍

**Comment by mheon** [comment]
LGTM

**Review by vrothberg — APPROVED** [review]
/lgtm
/hold

Restarted the flaked job.

**Review by flouthoc — APPROVED** [review]
LGTM
/hold cancel

**Pull request merged**
